src/ProductConstructionService/ProductConstructionService.Api/Api/v2018_07_16/Controllers/SubscriptionsController.cs

```diff
@@ -181,12 +185,22 @@ where sub.Enabled
 
         if (subscriptionToUpdate != null)
         {
+            _logger.LogInformation("Will trigger {subscriptionId} with build {buildId}", subscriptionId, buildId);
+
             await _workItemProducerFactory.CreateProducer<SubscriptionTriggerWorkItem>(subscriptionToUpdate.SourceEnabled).ProduceWorkItemAsync(new()
             {
                 SubscriptionId = subscriptionToUpdate.Id,
                 BuildId = buildId
             });
         }
+        else if (buildId != 0)
+        {
+            _logger.LogInformation("Suitable build {buildId} was not found in channels matching subscription {subscriptionId}. Not triggering updates", buildId, subscriptionId);
```

Comment:
```suggestion
            _logger.LogInformation("Suitable build {buildId} was not found in channel matching subscription {subscriptionId}. Not triggering updates", buildId, subscriptionId);
```
maybe we can even say which channel it was

Reply:
We don't have that information here but it's easy to get from the subscription